lib/warren/app/config.rb

```diff
@@ -7,13 +7,18 @@ class Config
       # We keep the template as plain text as it allows us to add comments
       TEMPLATE = <<~TEMPLATE
```

Comment:
Ideally I'd move this out into a template file like I did with the subscriber class files.